packages/connector-smtp/src/index.ts

```diff
@@ -32,22 +33,15 @@ const sendMessage =
       )
     );
 
-    const configOptions: SMTPTransport.Options = {
-      host,
-      port,
-      auth: {
-        user: username,
-        pass: password,
-      },
-      // Set `secure` to be false and `requireTLS` to be true to make sure `nodemailer` calls STARTTLS, which is wildly adopted in email servers.
-      secure: false,
-      requireTLS: true,
-      // Enable `logger` to help debugging.
-      logger: true,
-    };
+    const configOptions: SMTPTransport.Options = config;
 
     const transporter = nodemailer.createTransport(configOptions);
 
+    // See https://nodemailer.com/smtp/proxies/#2-using-socks-proxy.
+    if (config.proxy?.startsWith('socks')) {
+      transporter.set('proxy_socks_module', socks);
```

Comment:
Wondering is it properly to add this proxy configs?

Reply:
I followed the guides of nodemailer and tested locally, it worked fine.
<img width="1334" alt="image" src="https://user-images.githubusercontent.com/15182327/195490573-c08e2d4c-3368-4fc5-853a-a675ae4df863.png">

Reply:
My point is that do we need to implement the "proxy" part.

Reply:
@logto-io/eng